app/controllers/users_controller.rb

```diff
@@ -27,19 +27,61 @@ def create
     end
   end
 
-  def update
+  def edit
+    @genres = Genre.all
+    @musician_skill = MusicianSkill.all
+    @user = current_user
+    @id = current_user.id
+    @image = current_user.image
   end
 
-  def destroy
```

Comment:
how to remove account without this action?

Reply:
Devise